templates/home/reports/pre-a11y-1-en.html

```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Pre-accessibility assessment #1",
+	"language": "en",
+	"description": "Assessing the main content of the GC Home Page template",
+	"tag": "home",
+	"parentdir": "home",
+	"parentPage": "Home - Canada.ca",
+	"parentPageURL": "home-en.html",
+	"altLangPage": "a11y-1-fr.html",
```

Comment:
This page don't exists

```suggestion
	"altLangPage": "pre-a11y-1-fr.html",
```